lib/puma/plugin.rb

```diff
@@ -109,7 +109,7 @@ def in_background(&blk)
     end
 
     def workers_supported?
-      return false if Puma.jruby? || Puma.windows?
+      return false if Puma.jruby? || Puma.windows? || Puma.truffle?
```

Comment:
should we check ::Process.respond_to?(:fork) as well?
Does that single check eliminate the need to check for jruby/windows/truffle?

Reply:
@nateberkopec

Reply:
Fixed in my nitpick commit. Thanks for the work on this PR!

Reply:
Sweet!